docs/pages/deployment/docker.rst

```diff
@@ -0,0 +1,129 @@
+.. _running-docker:
+
+Running on Docker
+#################
+
+If you already use Docker, the easiest way to get your Nuts Node up and running for development or production is
+using Docker. This guide helps you to configure the Nuts node in Docker.
+To use the most recent release use ``nutsfoundation/nuts-node:latest``. For production environments it's advised to use a specific version.
+
+User
+****
+
+The default user in the container is ``18081`` that is only part of group ``18081``.
```

Comment:
Should we note this is not relevant when not mounting the data dir (e.g. only `did:web` DID and storage in Postgres/Mysql)?